itest/rust/src/variant_test.rs

```diff
@@ -334,6 +334,73 @@ fn variant_type_correct() {
     );
 }
 
+#[itest]
+fn variant_stringify_correct() {
+    use godot::builtin::dict;
+
+    assert_eq!("value".to_variant().stringify(), gstr("value"));
+    assert_eq!(Variant::nil().stringify(), gstr("<null>"));
+    assert_eq!(true.to_variant().stringify(), gstr("true"));
+    assert_eq!(30.to_variant().stringify(), gstr("30"));
+    assert_eq!(
+        godot::builtin::varray![1, "hello", false]
+            .to_variant()
+            .stringify(),
+        gstr("[1, \"hello\", false]")
+    );
+    assert_eq!(
+        dict! { "KEY": 50 }.to_variant().stringify(),
+        gstr("{ \"KEY\": 50 }")
+    );
+}
+
+#[itest]
+fn variant_booleanize_correct() {
+    use godot::builtin::dict;
+    use godot::builtin::varray;
```

Comment:
You can import those two at the beginning of the file, then it makes some code easier.